### ISSUE TITLE
Livestreamer package not found in Ubuntu 14.04

### ISSUE THREAD

**Issue by Maccer1** [chrippa/livestreamer#1220]
I can install any other package but apt-get is unable to find livestreamer. How can I install it?

**Comment by skulblakka** [comment]
While this is not really an answer to your problem I would recommend that you install livestreamer using pip (I would recommend the development version because the last release is kinda old). Take a look at the [documentation](http://docs.livestreamer.io/install.html#source-code) for more details.

**Comment by Maccer1** [comment]
Yeah. I ended up doing that. But I hope they fix the missing package or edit the doc because it clearly says I can install it with apt-get.
